### PR TITLE
chore: update docker-compose example and add scrape target example

### PR DESCRIPTION
**Pull request by ShivamB25**
**Description of the change**
- The URLs for Sonarr, Radarr, Lidarr, Prowlarr, and Sabnzbd have been updated to use service names instead of IP addresses. This allows accessing the services on the same network.
- The container names for Lidarr, Prowlarr, and Sabnzbd have been added.
- The restart policy has been set to "unless-stopped" for all services.

🆕 feat(config): Add Prometheus configuration file

- Added a new Prometheus configuration file to enable scraping metrics from the services.
- Renamed job names in the configuration file for clarity.
- Configured Prometheus to scrape metrics from Sonarr, Radarr, Prowlarr, Sabnzbd, and Lidarr exporter services.

ℹ️ Remember to configure the custom network if needed for accessing the services on the same network.

**Applicable issues**

- fixes #

**Additional information**

### CODE REVIEW

**Comment by ShivamB25** [comment]
just a little bit of edit to make it look better in repo. thanks for your work

File: examples/compose/docker-compose.yaml [hunk 1]
```diff
@@ -7,26 +7,64 @@ services:
     command: ["sonarr"]
     environment:
       PORT: 9707
-      URL: "http://x.x.x.x:8989"
+      URL: "http://x.x.x.x:8989" ## service name can also be used if they are on the same network for example http://sonarr:8989
```
Comment (onedr0p) [inline examples/compose/docker-compose.yaml:10]:
```suggestion
      URL: "http://x.x.x.x:8989" # or; http://sonarr:8989
```

File: examples/compose/docker-compose.yaml [hunk 2]
```diff
@@ -7,26 +7,64 @@ services:
...
       APIKEY: "xxx"
+#    networks:
+#    - your_custom_network  ## optional unless to access on the same network.
```
Comment (onedr0p) [inline examples/compose/docker-compose.yaml:13]:
```suggestion
#    - your_custom_network # optional
```

File: examples/compose/docker-compose.yaml [hunk 3]
```diff
@@ -7,26 +7,64 @@ services:
...
       - "9707:9707"
+    restart: unless-stopped
   radarr-exporter:
     image: ghcr.io/onedr0p/exportarr:v1.0.0
     container_name: radarr-exporter
     command: ["radarr"]
     environment:
       PORT: 9708
-      URL: "http://x.x.x.x:7878"
+      URL: "http://x.x.x.x:7878" ## service name can also be used if they are on the same network for example http://radarr:7878
```
Comment (onedr0p) [inline examples/compose/docker-compose.yaml:23]:
```suggestion
      URL: "http://x.x.x.x:7878" # or; http://radarr:7878
```

File: examples/compose/docker-compose.yaml [hunk 4]
```diff
@@ -7,26 +7,64 @@ services:
...
       APIKEY: "xxx"
+#    networks:
+#    - your_custom_network  ## optional unless to access on the same network.
```
Comment (onedr0p) [inline examples/compose/docker-compose.yaml:26]:
```suggestion
#    - your_custom_network # optional
```

File: examples/compose/docker-compose.yaml [hunk 5]
```diff
@@ -7,26 +7,64 @@ services:
...
       - "9708:9708"
+    restart: unless-stopped
   lidarr-exporter:
     image: ghcr.io/onedr0p/exportarr:v1.0.0
+    container_name: lidarr-exporter
     command: ["lidarr"]
     environment:
       PORT: 9709
-      URL: "http://x.x.x.x:8686"
+      URL: "http://x.x.x.x:8686" ## service name can also be used if they are on the same network for example http://lidarr:8686
```
Comment (onedr0p) [inline examples/compose/docker-compose.yaml:36]:
```suggestion
      URL: "http://x.x.x.x:8686" # or; http://lidarr:8686
```

File: examples/compose/docker-compose.yaml [hunk 6]
```diff
@@ -7,26 +7,64 @@ services:
...
       APIKEY: "xxx"
+#    networks:
+#     - your_custom_network  ## optional unless to access on the same network.
```
Comment (onedr0p) [inline examples/compose/docker-compose.yaml:39]:
```suggestion
#     - your_custom_network # optional
```

File: examples/compose/docker-compose.yaml [hunk 7]
```diff
@@ -7,26 +7,64 @@ services:
...
     ports:
       - "9709:9709"
+    restart: unless-stopped
+  prowlarr-exporter:
+    image: ghcr.io/onedr0p/exportarr:latest
+    container_name: prowlarr-exporter
+    command: ["prowlarr"]
+    environment:
+      PORT: 9710
+      URL: "http://x.x.x.x:9696" ## service name can also be used if they are on the same network for example http://prowlarr:8080
```
Comment (onedr0p) [inline examples/compose/docker-compose.yaml:49]:
```suggestion
      URL: "http://x.x.x.x:9696" # or; http://prowlarr:8080
```

File: examples/compose/docker-compose.yaml [hunk 8]
```diff
@@ -7,26 +7,64 @@ services:
...
+      APIKEY: "abc"
+      # PROWLARR_BACKFILL: true ##optional but recommended
+      # PROWLARR_BACKFILL_SINCE_DATE: "2023-03-01" ##optional but recommended
+#    networks:
+#     - your_custom_network  ## optional unless to access on the same network.
```
Comment (onedr0p) [inline examples/compose/docker-compose.yaml:54]:
```suggestion
      # PROWLARR_BACKFILL: true # optional
      # PROWLARR_BACKFILL_SINCE_DATE: "2023-03-01" # optional
#    networks:
#     - your_custom_network # optional
```

File: examples/compose/docker-compose.yaml [hunk 9]
```diff
@@ -7,26 +7,64 @@ services:
...
+  sabnzbd-exporter:
+    image: ghcr.io/onedr0p/exportarr:latest
+    container_name: sabnzbd-exporter
+    command: ["sabnzbd"]
+    environment:
+      PORT: 9711
+      URL: "http://x.x.x.x:8080" ## service name can also be used if they are on the same network for example http://sabnzbd:8080
+      APIKEY: "abc"
+#    networks:
+#     - your_custom_network  ## optional unless to access on the same network.
```
Comment (onedr0p) [inline examples/compose/docker-compose.yaml:67]:
```suggestion
      URL: "http://x.x.x.x:8080" # or; http://sabnzbd:8080
      APIKEY: "abc"
#    networks:
#     - your_custom_network # optional
```

File: examples/compose/docker-compose.yaml [hunk 10]
```diff
@@ -7,26 +7,64 @@ services:
...
+    ports:
+     - "9711:9711"
+    restart: unless-stopped
```
Comment (onedr0p) [inline examples/compose/docker-compose.yaml:70]:
```suggestion
    restart: unless-stopped

```

File: examples/prometheus/prometheus.yml [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+    static_configs:
+      - targets: ["prowlarr-exporter:9709"]  # Use service name and port
+
+  - job_name: "sabnzbd-exporter"  # Renamed job_name for clarity
+    static_configs:
+      - targets: ["sabnzbd-exporter:9710"]  # Use service name and port
+
+  - job_name: "lidarr--exporter"  # Renamed job_name for clarity
+    static_configs:
+      - targets: ["lidarr--exporter:9711"]  # Use service name and port
```
Comment (onedr0p) [inline examples/prometheus/prometheus.yml:27]:
```suggestion
  - job_name: "lidarr-exporter"  # Renamed job_name for clarity
    static_configs:
      - targets: ["lidarr-exporter:9711"]  # Use service name and port

```

**Review by onedr0p — CHANGES_REQUESTED** [review]

**Comment by ShivamB25** [comment]
@onedr0p wouldn't it be better to use latest while on compose , it is using v1.0.0 while i believe current one is at 1.5+

**Comment by onedr0p** [comment]
No, I do not want people to copy and paste and use the latest tag. I hope people are smart enough to update the version seeing how these are just examples.

**Comment by ShivamB25** [comment]
i see alright @onedr0p still would like it to bumped to atleast use prowlarr ❤️

**Comment by onedr0p** [comment]
Feel free to open a PR to at least get it on the most recent version :)

**Pull request merged**